packages/core/src/lib/TwaGenerator.ts

```diff
@@ -250,16 +250,23 @@ export class TwaGenerator {
     }));
   }
 
-  private async writeWebManifest(webManifestUrl: URL, targetDirectory: string): Promise<void> {
-    const response = await fetch(webManifestUrl);
+  private async writeWebManifest(twaManifest: TwaManifest, targetDirectory: string): Promise<void> {
```

Comment:
I'm not quite sure what's going on with the typing here - how have you changed the first parameter of this method from a URL to a TwaManifest but you haven't changed any of the callers?

Reply:
IIRC, this is only called here: https://github.com/GoogleChromeLabs/bubblewrap/pull/440/files#diff-6020d7a7812abc3186bdd457af35e210e130e887a016dfcbdfffe9b2e5fbc304R403 and the line invoking this was changed from `this.writeWebManifest(twaManifest.webManifestUrl, targetDirectory);` to `this.writeWebManifest(twaManifest, targetDirectory);`

Reply:
Oh right, sorry, I didn't twig that line 403 was in a different method that called this one. (I didn't see the GitHub code collapsing.)